app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -1,8 +1,49 @@
 import React, { useContext } from 'react';
 import UploadContext from '../context/upload';
 
+/**
+ * Returns a promise resolving to an ArrayBuffer representation of the given Blob object.
+ *
+ * @param {Blob} blob Blob object.
+ *
+ * @return {Promise<DataView>}
+ */
+export function blobToDataView(blob) {
```

Comment:
Couple notes on this:

1. [`blob.arrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/API/Blob/arrayBuffer) would be much more convenient, but [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Blob/arrayBuffer#Browser_compatibility) is a concern.
2. This could resolve to `ArrayBuffer`, but `DataView` is used, since I found that in the test environment, the mocked `@peculiar/webcrypto` webcrypto doesn't interoperate well with the mocked `jsdom` `FileReader` result ([throws](https://github.com/PeculiarVentures/pvtsutils/blob/922b3bd478014cbead96f034897792128131ff97/src/buffer_source_converter.ts#L21)).